idaes/models_extra/gas_solid_contactors/unit_models/tests/test_FB0D.py

```diff
@@ -176,6 +179,179 @@ def test_build(self, iron_oc):
     def test_dof(self, iron_oc):
         assert degrees_of_freedom(iron_oc) == 0
 
+    @pytest.fixture(scope="class")
+    def iron_oc_unscaled(self):
+        m = ConcreteModel()
+        m.fs = FlowsheetBlock(
+            default={"dynamic": True, "time_set": [0, 3600], "time_units": pyunits.s}
+        )
+
+        m.fs.gas_props = GasPhaseParameterBlock()
+        m.fs.solid_props = SolidPhaseParameterBlock()
+        m.fs.solid_rxns = HeteroReactionParameterBlock(
+            default={
+                "solid_property_package": m.fs.solid_props,
+                "gas_property_package": m.fs.gas_props,
+            }
+        )
+
+        m.fs.unit = FixedBed0D(
+            default={
+                "gas_property_package": m.fs.gas_props,
+                "solid_property_package": m.fs.solid_props,
+                "reaction_package": m.fs.solid_rxns,
+            }
+        )
+
+        # Discretize time domain
+        m.discretizer = TransformationFactory("dae.finite_difference")
+        m.discretizer.apply_to(m, nfe=100, wrt=m.fs.time, scheme="BACKWARD")
+
+        # Set reactor design conditions
+        m.fs.unit.bed_diameter.fix(1)  # diameter of the TGA reactor [m]
+        m.fs.unit.bed_height.fix(1)  # height of solids in the TGA reactor [m]
+
+        # Set initial conditions of the solid phase
+        m.fs.unit.solids[0].temperature.fix(1273.15)
+        m.fs.unit.solids[0].particle_porosity.fix(0.20)
+        m.fs.unit.solids[0].mass_frac_comp["Fe2O3"].fix(0.45)
+        m.fs.unit.solids[0].mass_frac_comp["Fe3O4"].fix(0)
+        m.fs.unit.solids[0].mass_frac_comp["Al2O3"].fix(0.55)
+
+        # Set conditions of the gas phase (this is all fixed as gas side
+        # assumption is excess gas flowrate which means all state variables
+        # remain unchanged)
+        for t in m.fs.time:
+            m.fs.unit.gas[t].temperature.fix(1273.15)
+            m.fs.unit.gas[t].pressure.fix(1.01325e5)  # 1atm
+            m.fs.unit.gas[t].mole_frac_comp["CO2"].fix(0.4)
+            m.fs.unit.gas[t].mole_frac_comp["H2O"].fix(0.5)
+            m.fs.unit.gas[t].mole_frac_comp["CH4"].fix(0.1)
+
+        return m
+
+    @pytest.mark.solver
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.component
+    def test_initialize_unscaled(self, iron_oc_unscaled):
+        optarg = {
+            "bound_push": 1e-8,
+            "halt_on_ampl_error": "yes",
+            "linear_solver": "ma27",
+        }
+
+        initialization_tester(iron_oc_unscaled)
+
+        solver = get_solver("ipopt", optarg)  # create solver
+
+        initialize_by_time_element(iron_oc_unscaled.fs,
```

Comment:
First, this looks like it should be separate from the `initialization_tester` above.

Secondly, there should be some assert statement following this to ensure that it actually did what was intended.

I think this also applies to some other similar tests later in this file.